recipes/libsvm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "3.21" %}
+{% set flatversion = version.replace(".", "") %}
+
+package:
+  name: libsvm
+  version: {{ version }}
+
+source:
+  fn: libsvm-{{ flatversion }}.tar.gz
+  url: https://github.com/cjlin1/libsvm/archive/v{{ flatversion }}.tar.gz
+  md5: acff2c058b7cb38a135fa5d0b4f39e43
+
+  patches:
+    - win32.patch  # [win32]
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+
+test:
+  requires:
+    - python  # [win]
```

Comment:
If we can switch back to using `environ['PY_VER']` here, but leave everything else alone (including build requirements) and CI passes. I would happily merge this. Thanks for trying this out here so that we can see this doesn't work.